src/transactions/InvokeHostFunctionOpFrame.cpp

```diff
@@ -346,6 +407,7 @@ InvokeHostFunctionOpFrame::doApply(Application& app, AbstractLedgerTxn& ltx,
     HostFunctionMetrics metrics(app.getMetrics());
     auto const& sorobanConfig =
         app.getLedgerManager().getSorobanNetworkConfig();
+    metrics.noteLimits(sorobanConfig);
```

Comment:
This is not the proper place for monitoring this. We shouldn't emit config more than once per ledger. On the other hand, if a ledger has 0 invoke host fn ops, we would skip this. We should monitor this somewhere during the ledger close e.g. in the ledger manager.

Reply:
Makes sense, I moved it to `closeLedger` in the ledger manager.